app/controllers/my_facilities/drug_stocks_controller.rb

```diff
@@ -28,7 +28,7 @@ def new
 
   def create
     DrugStocksCreator.call(current_admin, @facility, @for_end_of_month, drug_stocks_params[:drug_stocks])
-    redirect_to redirect_url(bust_cache: true), notice: "Saved drug stocks"
+    redirect_to redirect_url, notice: "Saved drug stocks"
```

Comment:
We shouldn't be using the cache busting query string in our application logic, as its brittle and also will expose it regular users.  I've updated the cache key for drug stocks to get invalidated when new drug stocks are created, which decouples it from any sort of controller logic.